### PR TITLE
fix(npm2yarn): use forked npm-to-yarn; fix a few conversion bugs

### LINKED ISSUE #5861: [npm2yarn]: `npm init` => `yarn create`
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#reporting-new-issues).

### Prerequisites

- [X] I'm using the latest version of Docusaurus.
- [X] I have tried the `npm run clear` or `yarn clear` command.
- [X] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages.
- [X] I have tried creating a repro with https://new.docusaurus.io.
- [X] I have read the console error message carefully (if applicable).

### Description

See demo here: https://du0ui.sse.codesandbox.io/docs/intro

If you use `@docusaurus/remark-plugin-npm2yarn` and your code snippet is set like this:

    ```sh npm2yarn
    npm init wdio ./foo
    ```

I would expect that it would transform the yarn version to `yarn create wdio ./foo`.

### Steps to reproduce

1. create a standard docusarus page
2. add `@docusaurus/remark-plugin-npm2yarn` plugin
3. add example as mention above

### Expected behavior

I would expect that it would transform the yarn version to `yarn create wdio ./foo`.

### Actual behavior

It displays `npm init wdio ./foo` and `yarn init wdio ./foo` which results in the attempt of creating a new yarn package rather than running the `create-wdio` starter package.

### Your environment

- Public source code:
- Public site URL:
- Docusaurus version used:
- Environment name and version (e.g. Chrome 89, Node.js 16.4):
- Operating system and version (e.g. Ubuntu 20.04.2 LTS):

### Reproducible demo

https://codesandbox.io/s/nice-fire-du0ui?file=/docusaurus.config.js

### Self-service

- [x] I'd be willing to fix this bug myself.

### PR DESCRIPTION
**Pull request by armano2**
## Motivation

I decided to fork `npm-to-yarn` package and add support for missing commands, 

- `yarn create name` -> `npm init name` and `yarn init name` -> `npm create name`
- `npm run x -- --args` -> `yarn x --args` or `yarn run x --args`
- `npm ls abc bcd` -> `yarn list --pattern "abc|bcd"`
- `npm test/start/stop`  -> `yarn test/start/stop`

see: 
- https://typescript-eslint.io/docs/linting/troubleshooting#how-do-i-check-to-see-what-versions-are-installed
- https://www.electronjs.org/docs/latest/tutorial/quick-start#create-your-application

fixes #5861

## Note

There is pr for original package at https://github.com/nebrelbug/npm-to-yarn/pull/22 

i did more small fixes at https://github.com/armano2/npm-to-yarn/commits/main, mainly https://github.com/armano2/npm-to-yarn/commit/2b9198390fec46dcb56c9c8f92ac5f81358d5437

## Test plans

https://deploy-preview-6005--docusaurus-2.netlify.app/docs/next/i18n/tutorial
https://deploy-preview-6005--docusaurus-2.netlify.app/docs/i18n/crowdin
https://deploy-preview-6005--docusaurus-2.netlify.app/docs/i18n/git/#initialize-the-i18n-folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,13 +47,13 @@ Short on time? Check out our [5-minute tutorial ⏱️](https://tutorial.docusau
 
 Use the initialization cli to create your site:
 
-```bash
+```bash npm2yarn
```
Comment (Josh-Cena) [inline README.md:50]: You probably shouldn't modify `README` 🧐 It's not used on the website

Comment (armano2) [reply README.md:50]:
```suggestion
```bash
```

File: README.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ npm init docusaurus@latest [name] [template]
 
 Example:
 
-```bash
+```bash npm2yarn
```
Comment (armano2) [inline README.md:56]:
```suggestion
```bash
```

**Review by slorber — CHANGES_REQUESTED** [review]
Thanks for these useful contributions 💯 

FYI I'll more likely merge it as part of my own fork (as we'll likely move it to a docusaurus-core org someday) and publish it under `@docusaurus` npm org.

Will do that soon, for now I'm focusing on category indexes

**Comment by armano2** [comment]
any updates on this?

**Comment by Josh-Cena** [comment]
@armano2 Could you rebase first? We are in a pretty awkward situation right now because the @docusaurus organization is there but neither @slorber nor I have member access to it. Maybe we need to wait a bit more. No idea what FB politics is going on, but accepting forked code into our repo

**Comment by slorber** [comment]
I don't forget, but waiting for @zpao to give me member access. 

Will figure out if we should take temporary actions immediately (ie use a fork) until the org is set up properly

**Comment by bradzacher** [comment]
Is this something that we could just rewrite internally rather than forking externally?

If @armano2 has ideas for how the package can be rewritten entirely - it would save having to go through any hoops at all and could just be integrated into the plugin directly.

**Comment by Josh-Cena** [comment]
I don't know what's all the business around it, but I think we basically can't do anything meaningful about it😅

1. We can't copy the code to our own repo (although we used to do that a lot)
2. We _can_ fork the package under @slorber's account but he'd rather not at this stage

@bradzacher If TS-ESLint is open to fork the npm2yarn remark plugin, I can help facilitate that.

**Comment by armano2** [comment]
> @armano2 Could you rebase first?

rebased

**Comment by Josh-Cena** [comment]
Thanks @armano2 Sorry for stalling this PR longer than it should have😞 Not in my will

**Comment by armano2** [comment]
suppressed by #8650

File: packages/create-docusaurus/templates/shared/docs/intro.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ The command also installs all necessary dependencies you need to run Docusaurus.
 
 Run the development server:
 
-```bash
+```bash npm2yarn
```
Comment (armano2) [inline packages/create-docusaurus/templates/shared/docs/intro.md:38]: maybe we should mvoe this change to #8650

File: packages/create-docusaurus/templates/shared/docs/intro.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ Generate a new Docusaurus site using the **classic template**.
 
 The classic template will automatically be added to your project after you run the command:
 
-```bash
+```bash npm2yarn
```
Comment (armano2) [inline packages/create-docusaurus/templates/shared/docs/intro.md:26]: maybe we should mvoe this change to #8650

**Pull request closed without merging**